dotnet/src/IntegrationTests/PromptTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.PromptTemplate.Handlebars;
+using SemanticKernel.IntegrationTests.Connectors.OpenAI;
+using SemanticKernel.IntegrationTests.TestSettings;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SemanticKernel.IntegrationTests;
+
+public sealed class PromptTests : IDisposable
+{
+    public PromptTests(ITestOutputHelper output)
+    {
+        this._logger = new XunitLogger<Kernel>(output);
+        this._testOutputHelper = new RedirectOutput(output);
+        Console.SetOut(this._testOutputHelper);
+
+        // Load configuration
+        this._configuration = new ConfigurationBuilder()
+            .AddJsonFile(path: "testsettings.json", optional: false, reloadOnChange: true)
+            .AddJsonFile(path: "testsettings.development.json", optional: true, reloadOnChange: true)
```

Comment:
`development` is an identifier of application environment, and in our case we have settings for integration tests within only one requirement. Since `testsettings.development.json` doesn't exist, we can remove it:
```suggestion
```